### PR TITLE
[MNOE-1134] Organization name & description not appearing in Audit Log for subscription

### PR DESCRIPTION
**Pull request by yuritomanek**
@ouranos

### CODE REVIEW

File: core/app/models/mno_enterprise/audit_event.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def formatted_details
     def format_serialized_details
       AUDIT_LOG_CONFIG.fetch('events', {}).fetch(key, '') % metadata.symbolize_keys
     rescue KeyError => e
-      e.message
+      e.message unless key == 'subscription_update'
```
Comment (ouranos) [inline core/app/models/mno_enterprise/audit_event.rb]: Why did you need this?

Comment (yuritomanek) [reply core/app/models/mno_enterprise/audit_event.rb]: I did this as it was returning the message in the audit log and it looked messy.

**Comment by yuritomanek** [comment]
The `organisation_id` is being set in the `Subscription#to_audit_event` with the following line:
```
event[:organization_id] = relationships.organization&.dig('data', 'id') if relationships.respond_to?(:organization)
```
I'll need to dig further into it and see why its not being passed correctly.
